assets/javascript/apps.js

```diff
@@ -0,0 +1,97 @@
+var searchParam = "matthew McConoughey";
+var apiResponse;
+var actors = createOriginalActorsArray();
+
+function addButtons() {
+  if (localStorage.getItem("localStorageActors")) {
+    var actorsArr = JSON.parse(localStorage.getItem("localStorageActors"));
+  } else {
+    var actorsArr = createOriginalActorsArray();
+  }
```

Comment:
Interesting choice to set the initial actors in the local storage. Great application of the last few classes!